### PR TITLE
Adds redis up check to upstart conf

### PR DESCRIPTION
**Pull request by skim1420**
@spinnaker/google-reviewers

### CODE REVIEW

File: etc/init/spinnaker.conf [hunk 1]
```diff
@@ -4,22 +4,36 @@ start on filesystem or runlevel [2345]
...
+        if [ $i -eq 30 ]; then
+          echo "Cassandra is not up on localhost.
+          Start it, and then restart Spinnaker."
+        fi
+        sleep 1
+      done
     fi
+  fi
+
+  if ! nc -z localhost 6379; then
```
Comment (lwander) [inline etc/init/spinnaker.conf]: Once someone takes their redis instance and moves it to another node they will no longer be able to run spinnaker

File: etc/init/spinnaker.conf [hunk 2]
```diff
@@ -31,3 +45,7 @@ pre-start script
     start $i
   done
 end script
+
+respawn
```
Comment (lwander) [inline etc/init/spinnaker.conf]: I don't think the top-level spinnaker upstart entry will be aware of the subservice health. This needs to go in orca's upstart.

Comment (skim1420) [inline etc/init/spinnaker.conf]: Yes. Added orca upstart conf ptal.

**Comment by skim1420** [comment]
Came full circle. Closing PR.

**Pull request closed without merging**